src/components/Timer/timer.js

```diff
@@ -94,6 +95,19 @@ export default class Timer extends Component {
             <span className="material-symbols-outlined">replay</span>
           </button>
         </div>
+        <div>
+          <div style={{ width: '100%', border: '1px solid #ccc', marginTop: '10px' }}>
```

Comment:
This is good for now. I'll then make sure I style this to fit the rest of the app.